installer/bundle/bundle_skel.sh

```diff
@@ -841,6 +842,19 @@ case "$installMode" in
         fi
 
         if [ $KIT_STATUS -eq 0 ]; then
+            # These steps are required to fix the mysql2 gem compatibility issue on latest Ubuntu distros and can be removed after GA release >2016_v1.2.0-75
+            if [ ${OMS_RUBY_DIR}/gem list | grep -q "mysql2" ]; then
+                # Cleans up static mysql2 ruby gem if it exists in omsagent ruby path
+                ${OMS_RUBY_DIR}/gem uninstall mysql2 1> /dev/null 2> /dev/null
+            fi
+			echo "----- Installing MySQL2 ruby gem -----"
+			if [ ! ${OMS_RUBY_DIR}/gem install mysql2 2>/dev/null ]; then
+                echo "MySQL ruby gem was not installed. MySQL ruby plugin would be disabled."
+                echo "Please check whether libmysqlclient dependency package is installed."
+                echo "You may need to 'apt-get install libmysqlclient-dev' or 'yum install mysql-devel',"
+                echo "then run '${{OMSHOME}}/ruby/bin/gem install mysql2' after omsagent installation to enable MySQL ruby plugin."
+            fi
+			
             # These conditionals exist due to some upgrade bugs in the postuninstall scripts of previous kits. These can be removed after GA.
             if [ -d /opt/microsoft/omsconfig ]; then
```

Comment:
I emailed JohnK about removing that segment , he confirmed that this workaround is no longer needed. However, it will require testing consistency invoker and since it is not related to this PR I intend to do that in a separate code review.